### PR TITLE
Add recipe for pytest-mock

### PR DESCRIPTION
**Pull request by nicoddemus**
cc @goanpeca

### CODE REVIEW

File: recipes/pytest-mock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{% set hash_type = "md5" %}
+{% set hash = "671c00dbbffee75d197eeec38c9bee91" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.python.org/packages/99/0e/45906c1e876b175cb51d8710075be900948f44a5f6a92c90095bdcd846c8/pytest-mock-{{ version }}.zip
```
Comment (ocefpaf) [inline recipes/pytest-mock/meta.yaml]: Use `https://pypi.io/packages/source/p/pytest-mock/pytest-mock-{{ version }}.zip` instead of the URL above.

Comment (nicoddemus) [inline recipes/pytest-mock/meta.yaml]: :+1:, thanks

**Comment by ocefpaf** [comment]
Good to go @nicoddemus?

**Comment by goanpeca** [comment]
Hi @nicoddemus, yes :-p ?

**Comment by nicoddemus** [comment]
@ocefpaf yes, thanks!

@goanpeca just in case you were interested in this package. :grin:

**Comment by goanpeca** [comment]
Hehe, have not used this one yet :-p

**Pull request merged**